gromacs/gromacs-2022.x/colvarproxy_gromacs.h

```diff
@@ -32,7 +32,7 @@ class colvarproxy_gromacs : public colvarproxy, public gmx::IForceProvider {
   bool colvars_restart;
   std::string config_file;
   size_t restart_frequency_s;
-  int previous_gmx_step;
+  int64_t previous_gmx_step;
```

Comment:
Why is `int64_t` used instead of `cvm::step_number`?

Reply:
I think this is harmless. If you remember, we typedef'ed `step_number` to equal `long long` before finding the guts to finally break with ancient C++ standards.

We may just as well use `int64_t` throughout now...

Reply:
@HanatoK Because that is the type of the data that we get from GROMACS.